### ISSUE TITLE
Classes not showing up on personal schedule if you have multiple persona

### ISSUE THREAD

**Issue by burlexpo** [bethlakshmi/GBE2#243]
If I'm on http://gbeadmin.webfactional.com/ (my "profile view"), my schedule (middle column) shows that I'm scheduled for The Rhinestone Revue on Friday (because The Boston Babydolls are, and I'm part of them.  All good) and a 'Stage Kitten' volunteer shift (because @jonkiparsky was demonstrating a point -- and we should probably remove me from that shift).

I can also see that I have three personae: The Boston Babydolls (my troupe); Scratch (my performer persona), and Mr. Scratch (my teacher persona).

I can also see that I submitted two classes, both of which have been accepted; neither one appears on my schedule.

If I look at http://gbeadmin.webfactional.com/scheduler/list, I can see my two classes: "The Care and Feeding of Volunteers" and "Getting the Most from Your Venue"; looking at the details, the teacher for both is "Mr. Scratch" (which is as it should be).

If I go and switch the teacher for one of the classes (Volunteers...) from "Mr. Scratch" to "Scratch", it shows up in my schedule.  The other one (where the teacher is still "Mr. Scratch") does not.

Since the whole purpose of this persona system was to allow someone to have multiple personae so they could have one bio for teaching and one for performing, it'd be nice if we had some way of making sure that everything showed up in one place.  I *may* be the only person using the functionality in this way, but there's no guarantee of that.

**Comment by bethlakshmi** [comment]
I got nervous, because I was the original author of schedule publication on the landing page... so I wanted to make sure I hadn't screwed up...

I went in and played with it and found the following:

1 - In resource allocations, Scratch had a persona allocated to his class.  Mr. Scratch did not have a resource allocation as a teacher for his class.

2 - I went into Getting the Most... and all I did was re-submit the scheduled event with all the same values.

3 - now it's all good.

@jonkiparsky  - in however we are allocating teachers - did we do the same thing in add a session as we do in edit...

I'm giving a quick look to answer my own question.  If I find something obvious, I will update... but since Scandal is setting these up as we speak, we may need a way to do data rehab.


FYI - I removed Scratch from Stage Kitten allocation... that worked fine.
